src/com/company/TAM1/Main.java

```diff
@@ -0,0 +1,39 @@
+package com.company.TAM1;
+
+import java.util.Arrays;
+
+public class Main {
+
+    public static final String SPECIFIED_CHARACTER = "w";
+
+    public static void main(String[] args) {
+        String sentence1 = "Width world Wide web";
+        String sentence2 = "WWW? Four-bedroom farmhouse in the countryside. Wave! All of the four double bedrooms are en suite.";
```

Comment:
This task is about working with file. Please add the reading data from the file. File examples present in the task description

Reply:
fixed